dev-docs/bidders/groupm.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: bidder
+title: GroupM
+description: Prebid GroupM Bidder Adaptor
+biddercode: groupm
+aliasCode: pubmatic
+media_types: banner, video, native
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+floors_supported: true
+userIds: all
+prebid_member: true
+safeframes_ok: true
+pbjs: true
+pbs: true
+pbs_app_supported: true
+fpd_supported: true
+gvl_id: 98
+---
+
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description        | Example                      | Type     |
+|---------------|----------|--------------------|------------------------------|----------|
+| `publisherId` | required | Publisher ID          | `'32572'`                 | `string` |
+| `adSlot`      | optional | Ad Slot Name (see below)| `'38519891'`            | `string` |
+| `pmzoneid`    | optional | Zone ID               | `'zone1,zone2'`           | `string` |
+| `lat`         | optional | Latitude              | `'40.712775'`             | `string` |
+| `lon`         | optional | Longitude             | `'-74.005973'`            | `string` |
```

Comment:
@pm-isha-bharti I see that the bidder params listed here don't exactly match what you have in Prebid Server. These docs are meant to communicate to the Publisher enabling `Groupm` as a bidder what parameters they need to pass to Prebid Server in the request for the `Groupm` bidder and so they should match what you have specified in the `static/bidder-params/groupm.json` file in the Prebid Server PR

Reply:
Hi @mansinahar The params that I have mentioned are supported at the prebid JS end and since `groupM` is an alias of `pubmatic`, I have kept the params similar to [pubmatic.md](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/pubmatic.md)
Should I add a separate section( "Prebid Server Test Request" ) for prebid-server request in the same document as I can see in other bidders docs as well?

Reply:
@pm-isha-bharti Ah ok! But I still see `wrapper` and `keywords` in the prebid server params file for Groupm but not in this doc. Can you please include those as well? And yeah, I think adding a test Prebid Server request is a good idea. Thanks!